client/python/gradio_client/client.py

```diff
@@ -327,7 +333,7 @@ def submit(
         inferred_fn_index = self._infer_fn_index(api_name, fn_index)
 
         helper = None
-        if self.endpoints[inferred_fn_index].use_ws:
+        if self.endpoints[inferred_fn_index].protocol in ("ws", "sse"):
```

Comment:
minor: let's modify and `use_ws` to check if the protocol is "ws" or "sse" since we're using it in different places

Reply:
sorry don't understand what you're suggesting

Reply:
Sorry, instead of `self.endpoints[inferred_fn_index].protocol in ("ws", "sse")` in multiple places, we do `self.endpoints[inferred_fn_index].use_http` or something like that.